typings/core-js.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for core-js v0.9.7
+declare type PropertyKey = string | number | symbol;
```

Comment:
Check if the file is obsolete when you installed latest version over `core-js` and `@types/core-js`, if so, please remove it.